src/EnergyPlus/OutputProcessor.cc

```diff
@@ -5875,7 +5879,7 @@ void UpdateDataandReport(EnergyPlusData &state, OutputProcessor::TimeStepType co
                 if (op->RVariableTypes(Loop).timeStepType != thisTimeStepType) continue;
                 auto &rVar(op->RVariableTypes(Loop).VarPtr);
                 // Update meters on the TimeStep  (Zone)
-                if (rVar.MeterArrayPtr != 0) {
+                if (rVar.MeterArrayPtr != 0 && !state.dataOutputProcessor->MeterValue.empty()) {
```

Comment:
I think we do need a new example file to be sure meter data reports correctly when only load calculations are performed.

Reply:
This feels a bit like a band-aid to me.  I am suspicious that we are just short-circuiting some piece of code that is necessary to setup MeterValue, and we should make sure to call that instead of adding this passive check here.  Maybe this is OK as-is, but I'll need to understand it a bit more.

Reply:
I agree with that.

Reply:
UpdateDataAndReport() is a big beast and it triggers tons of stuff to happen.  It is reused here to get output variables to generate during the ideal loads sizing calcs, but since it now can start getting called much earlier there are a lot of side effects that are not wanted until later, hence the band aids.    

Could make a smaller version of UpdateDataAndReport() to call during ideal loads sizing but that would duplicate code...